Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -313,9 +313,12 @@ private void RegisterServerEndpoints()
                 _logger.LogInformation("Registering publisher for {0} on {1}", fullService, address);
 
                 var uri = new UriBuilder(_appHost.GetSmartApiUrl(address.Address) + descriptorUri);
-                // DLNA will only work over http, so we must reset to http:// : {port}
-                uri.Scheme = "http://";
-                uri.Port = _netConfig.HttpServerPortNumber;
+                if (_appHost.PublishedServerUrl == null)
```

Comment:
This looks incomplete.  If my published server URL specifies a port, I don't see how it'll get picked up.

Reply:
Disregard - I was reading the diff, not the code in context...